app/jobs/flowdock_notification_job.rb

```diff
@@ -0,0 +1,8 @@
+class FlowdockNotificationJob  < ActiveJob::Base
+  queue_as :email
+
+  def perform(options)
+    @token = options
+    FlowdockNotification.otp(@token).deliver_later
+  end
+end
```

Comment:
@shivani I don't think we need to define a job for this. The ```deliver_later``` is creating the job for us in action_mailer. See http://api.rubyonrails.org/classes/ActionMailer/Base.html - it will do it for us so the following should just work

    FlowdockNotification.otp(@token).deliver_later

Right now, this is creating 2 jobs! One job to create another job that will send an email! :)